Formula/python3.rb

```diff
@@ -107,6 +107,9 @@ def install
     ENV["PYTHONHOME"] = nil
     ENV["PYTHONPATH"] = nil
 
+    ver = version.to_s.slice(/(3\.\d)/) || File.read("configure.ac").slice(/PYTHON_VERSION, (3\.\d)/, 1)
```

Comment:
configure.ac is always available during the build time (both stable release tarballs and mercurial repository has it), so I just drop ```version.to_s.slice(/(3\.\d)/)```